# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: df07abc82381
# Full Hash: df07abc82381d2a3067b7de63867a0181f497519
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-26 09:13:23
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r 5d39af040a61 -r df07abc82381 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Tue Mar 25 14:33:27 2025 +0000
+++ b/dom/ipc/ContentParent.h	Tue Mar 25 14:33:27 2025 +0000
@@ -99,6 +99,7 @@
 class TabContext;
 class GetFilesHelper;
 class MemoryReportRequestHost;
+class RemoteWorkerDebuggerManagerParent;
 class RemoteWorkerManager;
 class RemoteWorkerServiceParent;
 class ThreadsafeContentParentHandle;
@@ -1560,6 +1561,8 @@
 
   RefPtr<RemoteWorkerServiceParent> mRemoteWorkerServiceActor;
 
+  RefPtr<RemoteWorkerDebuggerManagerParent> mRemoteWorkerDebuggerManagerActor;
+
   UniquePtr<gfx::DriverCrashGuard> mDriverCrashGuard;
   UniquePtr<MemoryReportRequestHost> mMemoryReportRequest;
 